sql/src/test/java/io/crate/integrationtests/PostgresITest.java

```diff
@@ -514,6 +515,20 @@ public void testErrorRecoveryFromErrorsOutsideSqlOperations() throws Exception {
         }
     }
 
+    @Test
+    public void testErrorDetailsFromStackTraceInErrorResponse() throws Exception {
+        try (Connection conn = DriverManager.getConnection(JDBC_POSTGRESQL_URL, properties)) {
+            conn.setAutoCommit(true);
+            conn.createStatement().executeUpdate("create table t (i integer)");
+            ensureYellow();
+            conn.createStatement().executeUpdate("insert into t (i) values ('abc')");
+        } catch (PSQLException e) {
+            assertThat(e.getServerErrorMessage().getFile(), is("ValueNormalizer.java"));
```

Comment:
imho it's enough to check for notNull. This test shouldn't care about where the error is thrown but just verify that the information is there.